src/schematools/cli.py

```diff
@@ -511,91 +497,6 @@ def introspect_geojson(dataset_id: str, files: Iterable[str]) -> None:
     click.echo(json.dumps(aschema, indent=2))
 
 
-@import_.command("ndjson")
```

Comment:
Die eerste 3 imports (ndjson, geojson en events) zouden nog wel moeten blijven. De ndjson en geojson import worden daadwerkelijk gebruikt. Weliswaar via een API call naar schematools, maar het is wel nuttig om dit ook via de cli te kunnen aanroepen. En event gaat, als het goed is, ook nog gebruikt worden in de toekomst.

Reply:
Ok.